app-admin/prelude-manager/prelude-manager-3.0.0.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${PF}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tcpwrapper xml geoip dbx systemd"
+
+RDEPEND="net-libs/gnutls
+	dbx? ( dev-libs/libpreludedb )
+	tcpwrapper? ( sys-apps/tcp-wrappers )
+	xml? ( dev-libs/libxml2 )
+	geoip? ( dev-libs/libmaxminddb )"
+
```

Comment:
The changes between the original libev and the embedded one are negligible:
```
--- a/ev.h
+++ b/ev.h
@@ -554,11 +554,11 @@
 EV_API_DECL struct ev_loop *ev_default_loop_ptr;
 #endif

+extern struct ev_loop *ev_default_loop_ptr;
+
 EV_INLINE struct ev_loop *
 ev_default_loop_uc_ (void) EV_THROW
 {
-  extern struct ev_loop *ev_default_loop_ptr;
-  
   return ev_default_loop_ptr;
 }

--- a/Makefile.am
+++ b/Makefile.am
@@ -2,19 +2,12 @@

 VERSION_INFO = 4:0:0

-EXTRA_DIST = LICENSE Changes libev.m4 autogen.sh \
-            ev_vars.h ev_wrap.h \
-            ev_epoll.c ev_select.c ev_poll.c ev_kqueue.c ev_port.c ev_win32.c \
-            ev.3 ev.pod Symbols.ev Symbols.event
+EXTRA_DIST = LICENSE Changes libev.m4 ev_vars.h ev_wrap.h ev_epoll.c ev_select.c ev_poll.c ev_kqueue.c ev_port.c ev_win32.c

-man_MANS = ev.3
-
-include_HEADERS = ev.h ev++.h event.h
-
-lib_LTLIBRARIES = libev.la
-
-libev_la_SOURCES = ev.c event.c
+libev_la_SOURCES = ev.c
 libev_la_LDFLAGS = -version-info $(VERSION_INFO)

-ev.3: ev.pod
-       pod2man -n LIBEV -r "libev-$(VERSION)" -c "libev - high performance full featured event loop" -s3 <$< >$@
+noinst_HEADERS = ev.h ev_vars.h ev_wrap.h
+noinst_LTLIBRARIES = libev.la
+
+-include $(top_srcdir)/git.mk

```

I've created a pull request https://github.com/Prelude-SIEM/prelude-manager/pull/1 to get it fixed. Not sure if they take pull requests. I'll file a bug otherwise.